[2024-03-17T04:34:26.087+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage_dua.delete_dataset_csv_task manual__2024-03-17T04:34:11.166712+00:00 [queued]>
[2024-03-17T04:34:26.098+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: study_case_etl_cloud_storage_dua.delete_dataset_csv_task manual__2024-03-17T04:34:11.166712+00:00 [queued]>
[2024-03-17T04:34:26.099+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-17T04:34:26.115+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): delete_dataset_csv_task> on 2024-03-17 04:34:11.166712+00:00
[2024-03-17T04:34:26.121+0000] {standard_task_runner.py:60} INFO - Started process 4345 to run task
[2024-03-17T04:34:26.124+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'study_case_etl_cloud_storage_dua', 'delete_dataset_csv_task', 'manual__2024-03-17T04:34:11.166712+00:00', '--job-id', '899', '--raw', '--subdir', 'DAGS_FOLDER/etl_day_bigquery/study_case_etl_cloud_storage_dua.py', '--cfg-path', '/tmp/tmpd65cerp_']
[2024-03-17T04:34:26.127+0000] {standard_task_runner.py:88} INFO - Job 899: Subtask delete_dataset_csv_task
[2024-03-17T04:34:26.173+0000] {task_command.py:423} INFO - Running <TaskInstance: study_case_etl_cloud_storage_dua.delete_dataset_csv_task manual__2024-03-17T04:34:11.166712+00:00 [running]> on host 1d5b76b26ba5
[2024-03-17T04:34:26.242+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Agus mahari' AIRFLOW_CTX_DAG_ID='study_case_etl_cloud_storage_dua' AIRFLOW_CTX_TASK_ID='delete_dataset_csv_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-17T04:34:11.166712+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-17T04:34:11.166712+00:00'
[2024-03-17T04:34:26.245+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-17T04:34:26.245+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -rf /opt/***/data/fetch_data_task_id_2024-03-17.csv']
[2024-03-17T04:34:26.257+0000] {subprocess.py:86} INFO - Output:
[2024-03-17T04:34:26.261+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-17T04:34:26.283+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=study_case_etl_cloud_storage_dua, task_id=delete_dataset_csv_task, execution_date=20240317T043411, start_date=20240317T043426, end_date=20240317T043426
[2024-03-17T04:34:26.302+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-17T04:34:26.316+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
